### ISSUE TITLE
Advance search link doesn't work after initial advanced search

### ISSUE THREAD

**Issue by rkmase** [ldbase/ldbase_issues#160]
Reproduction instructions:

1. Go to a search page (https://www.ldbase.org/search?search_api_fulltext=)
2. "Go to Advanced search"
3. All: "testing", exclude: "behavior", Search
4. Results show, click on "Go to Advanced Search", search does not drop down

**Comment by barsenault-fsu** [comment]
There was an issue with the JavaScript in the theme.  Now on Test, if you try an Advanced Search, the results appear, the advanced search fields are visible, and the search terms are in the fields.

Once fixed, this raised an issue: if you were to click "Simple Search" after running an advanced search (sort of the opposite of the reported problem), the "advanced fields" still have values and get submitted even though they are hidden in the simple search.  In your reproduction instructions, "behavior" would still get submitted in the "exclude" field.

I first tried wiping the "advanced" fields if you click, "Go to Simple Search" but that seemed pretty destructive.  You couldn't get the terms back in the fields if you then tried to get back to advanced search.

Instead, it seems like once you start an advanced search, you'll probably want to stay in that mode or start a new search.  So now if you do an advanced search, I took away the "Go to Simple Search" link.
